README.md

```diff
@@ -4,36 +4,29 @@ A hackerspace is a physical place, run by people interested in various aspects o
 
 We are a breeding ground for awesome ideas. We provide a nest where those ideas can become reality. We operate by collaboration and openness. We allow people to fail, and to try again.
 
-# We failed, but we try again
+## We failed, but we try again
 
 We created our very own Ghent hackerspace. We had two rules: be excellent to each other and decide everything by consensus. We thought normal human interaction and common sense would solve all problems. Sadly, this was not true. When our hackerspace almost died, we decided to "Hack the Hackerspace". We found that the problems could all be traced to the following root causes:
 
- * We cannot rely on common sense because **people have different realities.**
- * **People have different, conflicting goals.** Because of that, consensus will never be reached on certain things. Problems will arise and they will not be solved. In most cases, no solution is worse than a bad solution.
+* We cannot rely on common sense because **people have different realities.**
+* **People have different, conflicting goals.** Because of that, consensus will never be reached on certain things. Problems will arise and they will not be solved. In most cases, no solution is worse than a bad solution.
 
 We knew that, in order to fix this, we needed a system that gets the best out of everyone and enables us to be awesome! After long late-night discussions, we came up with HTH. *This git repository contains the organizational structure of our hackerspace. This is intended to be place-agnostic information so it can be used by other hackerspaces around the world.*
 
-This is divided into 3 parts. Because [naming things is very hard](http://xkcd.com/910/) we decided to use the names from the ["Silo" series of Hugh Howey](http://en.wikipedia.org/wiki/Silo_%28series%29). The series follows the life of several people living in underground silo's in a post-apocalyptic society.
+This is divided into 3 parts.
+
+### [1. The System](./system)
 
-#### [1. The Order](https://github.com/0x20/HTH/tree/master/order)
 This is a description of the system that will run our hackerspace. You will find the different decision processes and the different entities of the organization.
 
 *In the series, the order is a book that describes how the silos should be managed. The order is clearly "bad" because all control is an infringement to the freedom of the people. Still, if you progress further in the series, you see that the order is still a good way to manage the people.*
```

Comment:
since the ref to the book was removed above, this should probably be removed too